connections/ethereum/connection.go

```diff
@@ -170,7 +170,7 @@ func (c *Connection) EstimateGasLondon(ctx context.Context, baseFee *big.Int) (*
 	var maxFeePerGas *big.Int
 
 	if c.maxGasPrice.Cmp(baseFee) < 0 {
-		maxPriorityFeePerGas = big.NewInt(1)
+		maxPriorityFeePerGas = big.NewInt(1000000000)
```

Comment:
can we make this configurable and by default `1000000000`?

Reply:
its possible and we will but for now its a quick fix